Jenkinsfile.deploy

```diff
@@ -205,7 +205,7 @@ pipeline {
                     }
                     post {
                         always {
-                            archiveArtifacts artifacts: 'test/logs/*/error */*.log', allowEmptyArchive: true
+                            archiveArtifacts artifacts: 'test/logs/**/*.log', allowEmptyArchive: true
```

Comment:
Set back storing all logs because I like also to see what is happening when there are no errors...
we could concentrate into 2 folders: `success` and `errors`